python/numpy/introduction_to_numpy.ipynb

```diff
@@ -670,16 +641,15 @@
    "source": [
     "https://numpy.org/doc/stable/reference/arrays.scalars.html#sized-aliases\n",
     "\n",
-    "* Integers: ``int32``, ``int64``, ``uint8`` ...\n",
-    "* Real numbers: ``float32``, ``float64`` ...\n",
-    "* Complex: ``complex64``, ``complex128``"
+    "* Integers: ``np.int32``, ``np.int64`` (python ``int``), ``np.uint8`` ...\n",
+    "* Real numbers: ``np.float32``, ``np.float64`` (python ``float``) ...\n",
+    "* Complex: ``np.complex64``, ``np.complex128`` (python ``complex``)"
```

Comment:
This I wouldn't add it. Especially because this is not that simple. Python int for example has a variable byte length.
Some types size might also depend on the platform...

Reply:
OK. Is this also tue for flot and complex ?
Anyway, I remove all refs to python built-in types.

Reply:
I think for float and complex most OS follow IEEE 754 standard. And complex are two float.
But if I remember well for double or long you can have some variety (windows vs linux).